framework/src/play/jobs/Job.java

```diff
@@ -164,7 +164,19 @@ public void every(String delay) {
      *            time in seconds
      */
     public void every(int seconds) {
-        JobsPlugin.executor.scheduleWithFixedDelay(this, seconds, seconds, TimeUnit.SECONDS);
+        // Wrap this job in a Runnable that catches and ignores all thrown exception.
+        // If there's an unhandled exception, then the executor would suppress all
+        // further executions, violating the contract.  Note that this does
+        // not interfere with the normal handling of onException().
+        Runnable neverThrowingRunnable = new Runnable() {
```

Comment:
> That commit, should have introduced the flag within now or a special now(boolean throwExecutionException):
45d390d#diff-34df15bef2557dbc31eb399824919cfeR138

my suggestion is:
* fix just the commit 45d390d on line 138 which introduced the error (by using a flag, check `now` and `in` too)
* for your own problems the use of `onException` should be sufficient

Or am I missing something? Overriding `onException` is the way to go if you want to ignore errors or special error handling.